CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+### Code style
+Regarding code style like indentation and whitespace, **follow the conventions you see used in the source already.**
+You can look at `.rubocop.yml` and `.codeclimate.yml`
+
+## Modifying the code
+First, ensure that you have installed Ruby 2.2+ and Postgresql
```

Comment:
Ruby > 2.3. We probably should lock that in a gemfile.

Reply:
Postgres min version? 9.5?

Reply:
Not sure about the Postgres version. So better keep it like this.